src/main/java/edu/ucsb/cs156/gauchoride/config/SecurityConfig.java

```diff
@@ -1,70 +1,103 @@
 import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
-import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityCustomizer;
+import org.springframework.security.config.annotation.web.configurers.oauth2.server.resource.OAuth2ResourceServerConfigurer;
+import org.springframework.security.config.http.SessionCreationPolicy;
+import org.springframework.security.core.Authentication;
 import org.springframework.security.core.GrantedAuthority;
 import org.springframework.security.core.authority.SimpleGrantedAuthority;
 import org.springframework.security.core.authority.mapping.GrantedAuthoritiesMapper;
 import org.springframework.security.oauth2.core.user.OAuth2UserAuthority;
+import org.springframework.security.web.SecurityFilterChain;
 import org.springframework.security.web.authentication.Http403ForbiddenEntryPoint;
+import org.springframework.security.web.authentication.SavedRequestAwareAuthenticationSuccessHandler;
+import org.springframework.security.web.authentication.www.BasicAuthenticationFilter;
 import org.springframework.security.web.csrf.CookieCsrfTokenRepository;
+import org.springframework.security.web.csrf.CsrfTokenRequestAttributeHandler;
+import org.springframework.security.web.csrf.CsrfTokenRequestHandler;
+import org.springframework.security.web.csrf.XorCsrfTokenRequestAttributeHandler;
+import org.springframework.security.web.csrf.CsrfToken;
 import org.springframework.security.web.util.matcher.AntPathRequestMatcher;
-import org.springframework.security.web.util.matcher.RequestMatcher;
-
-import edu.ucsb.cs156.gauchoride.entities.User;
-import edu.ucsb.cs156.gauchoride.repositories.UserRepository;
-import lombok.extern.slf4j.Slf4j;
+import org.springframework.util.StringUtils;
+import org.springframework.web.filter.OncePerRequestFilter;
 
 @Configuration
 @EnableWebSecurity
-@EnableGlobalMethodSecurity(prePostEnabled = true)
+@EnableMethodSecurity
 @Slf4j
-public class SecurityConfig extends WebSecurityConfigurerAdapter {
+public class SecurityConfig {
 
   @Value("${app.admin.emails}")
   private final List<String> adminEmails = new ArrayList<String>();
 
   @Autowired
   UserRepository userRepository;
 
-  @Override
-  protected void configure(HttpSecurity http) throws Exception {
-    http.authorizeRequests(authorize -> authorize
-        .anyRequest().permitAll())
-        .exceptionHandling(handlingConfigurer -> handlingConfigurer
-            .authenticationEntryPoint(new Http403ForbiddenEntryPoint()))
+  // @Override
+  // protected void configure(HttpSecurity http) throws Exception {
+  //   http.authorizeRequests(authorize -> authorize
+  //       .anyRequest().permitAll())
+  //       .exceptionHandling(handlingConfigurer -> handlingConfigurer
+  //           .authenticationEntryPoint(new Http403ForbiddenEntryPoint()))
```

Comment:
https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html

Reply:
Hi Professor Conrad, 

Sorry about that, I commented the code block out while I was adding changes. I have removed the comments.